roles/prometheus/tasks/cleanup.yml

```diff
@@ -1,8 +1,12 @@
 # code: language=ansible
 ---
+- name: Gather the service facts
```

Comment:
Yeah I think we could have it somewhere else if we need in other roles, otherwise we should just keep one of that per role. This will add the services vars to the hostvars from the time is executed and they can be referenced pretty anywhere

Reply:
Thinking a bit about this, how could we do it so we don't need to specify it in all 4 places? these belong to 2 different roles, and on each role there is a cleanup and the main task which I guess means we can't re-use any of the fact gathering, can we?

Reply:
> Thinking a bit about this, how could we do it so we don't need to specify it in all 4 places?

the most orthodox way of doing this would be:
- 1 task ensures firewalld is installed
- 1 task, which depends on the previous, ensures firewalld is running (check https://docs.ansible.com/ansible/latest/collections/ansible/builtin/service_module.html)
- all the other tasks needing to interact with the firewall depend on the latter.